Multisig/Logic/Models/Safe.swift

```diff
@@ -107,6 +107,9 @@ extension Safe {
         selection.safe = self
         App.shared.coreDataStack.saveContext()
         NotificationCenter.default.post(name: .selectedSafeChanged, object: nil)
+        
+        //TODO: This needs to be done in the right place
+        SafeDeploymentController.dismissNotification(safe: self)
```

Comment:
please remove it, user can just dismiss notification, I think we don't need to further complicate this